frontend/lib/screens/settings_screen.dart

```diff
@@ -16,7 +16,7 @@ class _SettingsScreenState extends State<SettingsScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
```

Comment:
The style of this app bar doesn't match those of the app bars in the rest of the app